### ISSUE TITLE
Some suggestions 

### ISSUE THREAD

**Issue by dhanvi** [ctfs/write-ups-2014#292]
Hi, @mathiasbynens 

The size of write-up's repo is growing more and more day by day which is not a good option for hosting it on github as it suggests to keep the repo under 1gb [source](https://help.github.com/articles/what-is-my-disk-quota/) and also for people who are trying to clone it as it will be dam slow; so why don't we create a new repo called write-up-2015 so that we can add all the ctf's done in 2015 to this repo so that it makes the write-ups more organised and we need not add the 2015 options in the end of every ctf name 

Also we can organise our repo as in [shell-storm](http://shell-storm.org/repo/CTF/CSAW-2014/) ie.. sort them using the category name even though it is mentioned in the readme file. So that it becomes easy for people who focus on a particular topic as one will not solve all the challenges of the ctf and he need not look through all the challenges under the ctf to see what he likes (as the name gives very less about the type of challenge)
Also inside the category of the challenge the challenge name will be ending with the score of the challenge which is also a good thing to follow :+1: 
Also one more good thing about the shell-storm is that it contains the score board :+1:  which increase the spirit of the ctf

So what do you say?? Hope these things will be implemented which makes this repo more organised and friendly to use. :smile:

**Comment by mathiasbynens** [comment]
https://github.com/ctfs/write-ups-2014
https://github.com/ctfs/write-ups-2015

**Comment by mathiasbynens** [comment]
https://github.com/blog/1986-announcing-git-large-file-storage-lfs would elegantly solve this problem without having to rely on a third-party host. +@technoweenie

The bad news:
> Every user and organization on GitHub.com with Git LFS enabled will begin with 1 GB of free file storage and a monthly bandwidth quota of 1 GB. If your workflow requires higher quotas, you can easily purchase more storage and bandwidth for your account.

**Comment by YASME-Tim** [comment]
I hope it is not enabled by default, because if so, we have to think of a (donation based) support. Especially with monthly bandwidth quota of 1GB

**Comment by dhanvi** [comment]
Or we could continue the present work flow to keep the files greater than 10mb out of the repo as we split the writeup by years and there is no way that the repo size will become greater than 1Gb a better option than paying a lot of money per month  

![screenshot from 2015-04-09 17 41 12](https://cloud.githubusercontent.com/assets/6671379/7066890/74d334a2-dee4-11e4-9cc3-19bcfc47a602.png)

we don't want to track all type of files of certain type as even some zip files will be having very less space that we need not track them using the git lfs 

also we usually have large files in forensics and they don't have a fixed file extension and we need to manually say them 

also I wish there is a way to configure every git lfs to track every file greater than 10 mb

**Comment by YASME-Tim** [comment]
We could use git hooks (haven't worked with them yet) to automatically add a `.gitignore` for files > 10MB, e.g. mentioned [here](http://stackoverflow.com/questions/29527405/git-warn-if-file-larger-than-a-pre-defined-size). I think the full year repos will be close to 1GB each year though - even with the concept to ignore all files >10MB.

**Comment by captn3m0** [comment]
Would be troublesome to implement. Most of our contributions come from web-based edits. Further, git-hooks are completely opt-in, ie they have to be setup on a system manually after cloning.

We could do it by fetching each PR, and then pruning it for large files, and then merging it, but it will be far more harder than the green button we have now.

**Comment by mathiasbynens** [comment]
> Most of our contributions come from web-based edits.

@captchaflag Yes, but ***no*** contributions that actually add challenge files (i.e. tarballs etc.) are web-based edits. (At least, I haven’t seen any)

> Further, git-hooks are completely opt-in, ie they have to be setup on a system manually after cloning.

This is true. We could add a simple Bash script to the repo that sets up the hooks.

**Comment by mathiasbynens** [comment]
For each pack of 50 GB of storage + 50 GB of bandwidth per month, it’s $5/month.

https://github.com/pricing#add-ons

**Comment by dhanvi** [comment]
well we may not be able to use it yet as the git clone is not pause and resume process so even if we use use the lfs it will be a problem with slow internet speed to clone the repo. Think of a case when you cloning stops at 95 %

**Comment by dhanvi** [comment]
well we may not be able to use it yet as the git clone is not pause and resume process so even if we use use the lfs it will be a problem with slow internet speed to clone the repo. Think of a case when you cloning stops at 95 %

**Comment by mathiasbynens** [comment]
The raw (non-git-lfs) Git repo size would be significantly smaller, though, and `git-lfs` would enable us to host the larger files as closely to the repo as possible (instead of manually uploading them to a third-party host and linking them).
